src/models/rsk/pegout-status-data-model.ts

```diff
@@ -19,6 +20,58 @@ export class PegoutStatusDataModel implements SearchableModel {
   feeInSatoshisToBePaid: number;
   status: PegoutStatus;
   btcRawTransaction: string;
+}
+
+export class PegoutStatusAppDataModel implements PegoutStatusDataModel{
+  constructor(data?: Partial<PegoutStatusAppDataModel>) {
+    Object.assign(this, data);
+  }
+
+  static fromPegoutStatusDbDataModel(model: PegoutStatusDbDataModel):PegoutStatusAppDataModel {
```

Comment:
does it need to be from `PegoutStatusDbDataModel` implementation or could it just be from `PegoutStatusDataModel` interface?